packages/odyssey-storybook/src/components/odyssey-mui/CircularProgress/CircularProgress.stories.tsx

```diff
@@ -38,13 +40,22 @@ const storybookMeta: Meta<CircularProgressProps> = {
 
 export default storybookMeta;
 
-
 export const Indeterminate: StoryObj<CircularProgressProps> = {
   args: {},
+  play: async ({ canvasElement }) => {
+    const canvas = within(canvasElement);
+    const el = await canvas.getByRole("progressbar");
+    await expect(el).toHaveAccessibleName();
+  },
```

Comment:
**NON-BLOCKING** What's it mean by accessible name? Can we check the value of the name?